kythe/go/platform/tools/kzip/mergecmd/mergecmd.go

```diff
@@ -90,7 +93,15 @@ func (c *mergeCommand) Execute(ctx context.Context, fs *flag.FlagSet, _ ...any)
 			vfs.Remove(ctx, tmpName)
 		}
 	}()
-	archives := fs.Args()
+	var archives []string
+	if c.inputFileList != "" {
```

Comment:
May allow `"-"` to mean stdin, so the following would work?

```kzip merge -output out.kzip -input_file_list -```

Reply:
:+1: done